apps/nextjs/src/components/tournaments/registrations-table.tsx

```
@@ -0,0 +1,47 @@
+  players: Player | null;
+}
+
+interface RegistrationsTableProps {
+  players: PlayerProfile[];
+}
+
+export default function RegistrationsTable({
+  players,
+}: Readonly<RegistrationsTableProps>) {
+  return (
+    <DataTable<PlayerProfile> data={players} columns={columns}></DataTable>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)